### PR TITLE
Add exclude folders option

### PR DESCRIPTION
**Pull request by goran1023**

### CODE REVIEW

File: src/tasks/sync.js [hunk 1]
```diff
@@ -45,11 +61,13 @@ const SyncSpaces = {
 
   async syncStories () {
     console.log(chalk.green('✓') + ' Syncing stories...')
-    var targetFolders = await this.client.getAll(`spaces/${this.targetSpaceId}/stories`, {
+    var _targetFolders = await this.client.getAll(`spaces/${this.targetSpaceId}/stories`, {
```
Comment (ademarCardoso) [inline src/tasks/sync.js:64]:
```suggestion
    const _targetFolders = await this.client.getAll(`spaces/${this.targetSpaceId}/stories`, {
```

File: src/cli.js [hunk 2]
```diff
@@ -169,13 +169,15 @@ program
   .requiredOption('--type <TYPE>', 'Define what will be sync. Can be components, folders, stories, datasources or roles')
   .requiredOption('--source <SPACE_ID>', 'Source space id')
   .requiredOption('--target <SPACE_ID>', 'Target space id')
+  .option('--exclude-folders', 'Define which folders (slugs) should be excluded from sync. Separate with comma')
```
Comment (ademarCardoso) [inline src/cli.js:172]:
```suggestion
  .option('-e, --exclude-folders <FOLDER_NAME>', 'Define which folders (slugs) should be excluded from sync. Separate with comma')
```

**Review by ademarCardoso — CHANGES_REQUESTED** [review]
Thank you very much for the contribution my friend, @goran1023 .

I did some tests and it didn't work well, I got some error messages I think the feature is interesting and I think we can improve the code a bit. I noticed that there were some iterations of other users in your PR which makes the feature relevant 😄 , iI will talk to the team to analyze the feasibility of the feature.

**Comment by lisilinhart** [comment]
Hi @goran1023 , would it be possible for you to reopen this PR  with the requested changes to  https://github.com/storyblok/storyblok-cli/? We're in the process of moving the code there and I can add your change by myself, but then your contribution would not be visible, so it would be better if you reopened it 🙂

**Pull request closed without merging**